### PR TITLE
Configurable Approvals

### LINKED ISSUE #186: Make validation requirements configurable
A future user should be able to setup easily, which users have to be approved and what consequences this has (eg what fields are visible with an without validation)

### PR DESCRIPTION
**Pull request by feeds**
- Make the approval configurable (= users are added to approved group on signup and fake user creation)
- Add a few new names instead of A here and there
- Fix the Signup

Partly fixes #186

### CODE REVIEW

File: backend/match4everyone/configuration/B.py [hunk 1]
```diff
@@ -13,8 +13,8 @@ class BConfig(ParticipantConfig):
     #    NewPermissions.can_contact_type_a_if_approved,
     # ]
 
-    profile_visible_for_A = True
-    profile_visible_for_other_B = False
+    profile_visible_for_participants_of_different_type = True
+    profile_visible_for_participants_of_same_type = False
```
Comment (bjrne) [inline backend/match4everyone/configuration/B.py:18]: Is this missing a
```suggestion
    needs_manual_approval_from_staff = True
```
line? :)

**Review by bjrne — APPROVED** [review]
Seems good!

File: backend/match4everyone/configuration/B.py (hunk 1 above)
Comment (feeds) [reply backend/match4everyone/configuration/B.py:18]: I set the default value to `True` in the superclass to make approvals the default behaviour, so it is not necessary.

Comment (bjrne) [reply backend/match4everyone/configuration/B.py:18]: I think I would still put it here for a better understanding where to change it, don't you think?

**Pull request merged**